### DISCUSSION TITLE
Docstring generator?

### DISCUSSION THREAD

**Discussion by nquaz** [jupyterlab/jupyterlab#16354] [General]
New to jupyter lab, have searched online if there are jupyter lab specific extensions that generate docstrings for .py files but can't find any. Are there any? Main reason I ask is because I would like to work on both .py and .ipynb within jupyter lab for a more streamlined experience-- it's rather cumbersome switching between jupyter lab's interface and other IDEs. I could also switch to another IDE altogether but I have not been satisfied with how some of them handle .ipynb

**Comment by vinayak19th** [comment]
I have the same issue
